apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -8,6 +10,7 @@ body {
 	min-width: 320px;
 	min-height: 100vh;
 	margin: 0;
+	overflow: hidden;
 	font-family: Arial, sans-serif;
```

Comment:
I think you forgot to add the main font (Nunito), this might violate rule 14 of quality criteria